.github/workflows/test.yaml

```diff
@@ -64,3 +64,21 @@ jobs:
       - name: Print workflow handler log
         run: bash -c "kubectl logs -n galaxy $(kubectl -n galaxy get pods | grep -o '[^ ]*galaxy-workflow[^ ]*')"
         if: always()
+      - name: Print service info
+        run: kubectl get svc -n galaxy
+        if: always()
+      - name: Print nginx service info
+        run: kubectl describe svc -n galaxy galaxy-nginx
+        if: always()
+      - name: Check appVersion
+        if: always()
```

Comment:
What does always() do btw? Do we need it here?

Reply:
The test will always run even if previous tests have failed.  It is likely not necessary here, and actually the two tests that do nothing by print the output of `kubectl` could be folded into the `appVersion` check.  We likely want the `appVersion` check to always run to catch as many problems as possible at once.